src/interface_adapter/chat/ChatState.java

```diff
@@ -0,0 +1,13 @@
+package interface_adapter.chat;
+
+import interface_adapter.logged_in.LoggedInState;
+
+public class ChatState {
+    public String message = "";
+
+    public ChatState(ChatState copy) {
```

Comment:
Should this just be message?

Reply:
It's possible that we may add more fields to ChatState in the future.
In addition, this is the paradigm established by our existing code.

Reply:
You're right... But it isn't used, so why can't we just do .clone?